src/gmp/z.d

```diff
@@ -562,6 +562,7 @@ nothrow:
             else static if (s == "|") __gmpz_ior(mut_rhs._ptr, _ptr, rhs._ptr);
             else static if (s == "^") __gmpz_xor(mut_rhs._ptr, _ptr, rhs._ptr);
             else static if (s == "<<") __gmpz_mul_2exp(mut_rhs._ptr, _ptr, cast(ulong)rhs);
+            else static if (s == ">>") __gmpz_tdiv_q_2exp(mut_rhs._ptr, _ptr, cast(ulong)rhs);
```

Comment:
We need to handle the cases where `rhs` is negative for both `<<` and `>>`, right?